core/domain/exp_domain.py

```diff
@@ -2384,8 +2366,31 @@ def from_yaml(cls, exploration_id, yaml_content):
         """
         exploration_dict = cls._migrate_to_latest_yaml_version(
             yaml_content, exploration_id)
+        exploration_dict = cls.add_id_and_proto_size_in_bytes_to_dict(
+            exploration_dict, exploration_id)
+
+        return Exploration.from_dict(exploration_dict)
+
+    def add_id_and_proto_size_in_bytes_to_dict(
+        cls, exploration_dict, exploration_id
+    ):
+        """Add id and proto_size_in_bytes to the exploration dict.
+
+        Args:
+            exploration_dict: dict. The dict representation of Exploration
+                object.
+            exploration_id: str. The exploration id.
+
+        Returns:
+            exploration_dict: dict. The updated dict representation of
+                Exploration object.
+        """
         exploration_dict['id'] = exploration_id
 
+        # Creating a exploration domain object to calcualte the
```

Comment:
"calculate" is misspelled (not sure if this happens anywhere else).

Reply:
Fixed, no it happens here only.